docs/modules/ROOT/pages/ollama.adoc

```diff
@@ -2,59 +2,29 @@
 
 include::./includes/attributes.adoc[]
 
-https://ollama.com/[Ollama] proposes a way to run large language models (LLMs) locally.
-You can run many https://ollama.com/library[models] such as LLama2, Mistral, or CodeLlama on your local machine.
+https://ollama.com/[Ollama] provides a way to run large language models (LLMs) locally.
+You can run many https://ollama.com/library[models] such as LLama3, Mistral, CodeLlama and many others on your machine, with full CPU and GPU support.
 
 [#_prerequisites]
 == Prerequisites
 
-To use Ollama, you need to have a running Ollama _server_.
-Go to the https://ollama.com/download[Ollama download page] and download the server for your platform.
+To use Ollama, you need to have a running Ollama installed.
+Ollama is available for all major platforms and its installation is quite easy, simply visit https://ollama.com/download[Ollama download page] and follow the instructions.
 
 Once installed, check that Ollama is running using:
 
 [source,shell]
 ----
-> ollama list
+> ollama --version
 ----
 
-It may not display any model, which is fine, let's pull the `llama2` model:
+=== Dev Service
 
-[source,shell]
-----
-> ollama pull llama2
-----
-
-WARNING: Models are huge. For example Llama2 is 3.8Gb. Make sure you have enough disk space.
-
-Let's also pull the _default_ embedding model:
+Quarkus LangChain4j automatically handles the pulling of the models configured by the application, so there is no need for users to do so manually.
```

Comment:
I think I'm missing something, but this doesn't work for me. 
I've removed `nomic-embed-text` from my ollama installation and tried running an app with

```
quarkus.langchain4j.devservices.enabled=true
quarkus.langchain4j.ollama.embedding-model.model-id=nomic-embed-text
```
and upon trying to create an embedding, I get `{"error":"model 'nomic-embed-text' not found, try pulling it first"}`

Anything else I need to do?

Reply:
None of those properties should be needed, but they should also not affect the outcome.

Are you sure you are using this locally built PR?

Reply:
Ah, sorry, I forgot to change the `quarkus-langchain4j.version` in the used sample to `999-SNAPSHOT`. This is going to happen many times before I get used to it.

Reply:
It's happened plenty to me as well 🤣